ui/components/src/TimeChart/TimeChart.tsx

```diff
@@ -227,9 +225,9 @@ export const TimeChart = forwardRef<ChartInstance, TimeChartProps>(function Time
       animation: false,
       tooltip: {
         show: true,
-        trigger: isStackedBar ? 'item' : 'axis',
-        // ECharts tooltip content hidden since we use custom tooltip instead
-        showContent: isStackedBar,
+        trigger: tooltipConfig.enableEChartsTooltip ? 'item' : 'axis',
```

Comment:
i think maybe we should still keep `isStackedBar`, otherwise it seems confusing why trigger is set to item when enableEChartsTooltip is true

Reply:
I can change back, we had another request internally before where I suggested using the ECharts tooltip with `__experimentalEChartsOptionsOverride` so thought having a more explicit way to do that could be useful (they only ever had at most 2 series and needed multi y axis support which we don't support yet)